Code/UnitTests/FoundationTest/Strings/UnicodeUtilsTest.cpp

```diff
@@ -1,4 +1,4 @@
-﻿#include <FoundationTestPCH.h>
+#include <FoundationTestPCH.h>
```

Comment:
The fact that this line is highlighted as "changed" most likely means that you lost the UTF-8 BOM. This always happens when we modify these files and then the tests fail.

If the file still contains Unicode letters, you need to use Visual Studio's "Save As" dialog, then "Save with Encoding" and use Utf-8 WITH BOM. Probably need to do this twice, because VS is "a little buggy" and often does it incorrectly the first time.